Server/GuestAppTestPlan.md

```diff
@@ -1,5 +1,7 @@
 ## 'Guest App' Test Plan
 
+- Run Testplan with encryption enabled and mixing browsers Firefox, Chrome, Safari, IE11 and Edge
```

Comment:
I would like to have specific scenarios for each browser. Just playing around with the UI.